uev.h

```diff
@@ -36,6 +36,9 @@
 #define UEV_WRITE       EPOLLOUT
 #define UEV_PRI         EPOLLPRI
 #define UEV_HUP         EPOLLHUP
+#define UEV_RDHUP       EPOLLRDHUP
+#define UEV_ET          EPOLLET
```

Comment:
I'd like to call this `UEV_EDGE`, that's not too long and slightly more descriptive than the Linux `EPOLLET`.